app/templates/posts.hbs

```diff
@@ -93,7 +93,7 @@
                 <li class="no-posts-box">
                     <div class="no-posts">
                         {{#if showingAll}}
-                            <h3>You Haven't Written Any Posts Yet!</h3>
+                            <h3>You haven't written any stories yet!</h3>
                             {{#link-to "editor.new"}}<button type="button" class="gh-btn gh-btn-green gh-btn-lg"><span>Write a new Post</span></button>{{/link-to}}
```

Comment:
I think the text in the button should be changed to 'Write a new story' as well?